nahand/main.js

```diff
@@ -6,7 +6,7 @@ let win
 
 function createWindow () {
   // Create the browser window.
-  win = new BrowserWindow({width: 800, height: 600})
+  win = new BrowserWindow({width: 1800, height: 1600})
```

Comment:
Is it possible to set that to the screen size? I wonder what contains the `screen` object. Might be stored there.